Parsers/Corelight/Corelight

```diff
@@ -0,0 +1,621 @@
+// Usage Instruction : 
```

Comment:
Generally we prefer these to have a .TXT extension for ease of offline usage.

Reply:
Ok, I`ll rename it